src/test/scala/uk/gov/hmrc/http/cache/client/ShortLivedCacheSpec.scala

```diff
@@ -142,7 +143,7 @@ object TestShortLiveHttpCaching extends ShortLivedHttpCaching {
       val cacheMap = new TestCacheMap(cacheId, Map(formId -> jsValue))
       map.put(cacheId, cacheMap)
       cacheMap
-    }
+    }(ec)
```

Comment:
why implicit ec but you pass it explicitly?

Reply:
Because for scala 2.12 without specifying execution context you have ambiguous implicit values

```
ambiguous implicit values:
[error]  both value ec in object TestShortLiveHttpCaching of type => scala.concurrent.ExecutionContext
[error]  and value executionContext of type scala.concurrent.ExecutionContext
[error]  match expected type scala.concurrent.ExecutionContext
[error]     Future {
```
And only this works for me in every case. Other combinations probably will be related with divided tests to different packages and making different implementation.

It's related with migration between scala 2.11 and 2.12 in Futures

Reply:
so just doing `import scala.concurrent.ExecutionContext.Implicits.global` at the very top of the file doesn't work? I didn't check if it does

Reply:
I can import it, but still in whole tests and object need to use explicit execution context (in this case with import I need to pass `global`)

Do you prefer this approach?

Reply:
yeah, I think I'd rather pass `global` or just `implicitly` where `ec` is passed now. Passing `global` may be more familiar for people

Reply:
Done, thanks! :)